src/test/java/pages/LogoutPopup.java

```diff
@@ -0,0 +1,26 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LogoutPopup {
+
+    private static final By LOGGED_IN_LINK = By.xpath(".//a[contains(@class, 'logedin')]");
+    private static final By LOGGED_OUT_BUTTON = By.xpath(".//a[contains(text(), 'Выйти')]");
+    private static final By LOGGED_OUT_RESULT = By.xpath(".//a[contains(text(), 'Войти')]");
+
+    private final WebDriver driver;
+
+    public LogoutPopup() {
+        this.driver = WebDriverSingleton.getInstance();
+    }
+
+    public void logout() {
```

Comment:
According to page object pattern, most of methods return current page or new page. It means, that if after logout you are on the same page, then method must return current page object, otherwise another. In this case after logout you are on another page, please return it (don't forget to update tests after that).